addons/crm/tests/test_crm_lead.py

```diff
@@ -370,6 +373,14 @@ def test_crm_lead_stages(self):
         self.assertEqual(lead.probability, 100.0)
         self.assertEqual(lead.stage_id, self.stage_gen_won)  # generic won stage has lower sequence than team won stage
 
+    @users('user_sales_leads')
+    @freeze_time("2012-01-14")
```

Comment:
Is freeze time required ?

Reply:
I guess it is to make the datetime.now() the same as date_closed in 
`self.assertEqual(self.lead_1.date_closed, datetime.now(), "Closed date is updated after marking lead as lost")`